src/App.js

```diff
@@ -1,9 +1,14 @@
-import React from 'react';
-import './App.css';
-import CounterExample from './CounterExample';
+import React, { useState } from 'react';
 
-function App() {
-  return <div>{<CounterExample />}</div>;
-}
+const App = () => {
```

Comment:
nice! it looks amazing,

you may try export default inline this component now 🥰

Reply:
How to export this component?

Reply:
inline -> that means in one line only, you can export default function without using `const`